wavespectra/input/spotter.py

```diff
@@ -6,16 +6,206 @@
+    ds['Latitude (deg)'].attrs['standard_name'] = 'latitude'
+    ds['Latitude (deg)'].attrs['long_name'] = 'Latitude'
+
+    ds['Longitude (deg)'].attrs['units'] = 'degree_east'
+    ds['Longitude (deg)'].attrs['standard_name'] = 'longitude'
+    ds['Longitude (deg)'].attrs['long_name'] = 'Longitude'
+
+    ds['Wind Speed (m/s)'].attrs['units'] = 'm/s'
+    ds['Wind Speed (m/s)'].attrs['standard_name'] = 'wind_speed'
+    ds['Wind Speed (m/s)'].attrs['long_name'] = 'Wind speed'
+
+    ds['Wind Direction (deg)'].attrs['units'] = 'degree'
+    ds['Wind Direction (deg)'].attrs['standard_name'] = 'wind_from_direction'
+    ds['Wind Direction (deg)'].attrs['long_name'] = 'Wind direction'
+
+    ds['Surface Temperature (°C)'] = 274.15*ds['Surface Temperature (°C)']
+    ds['Surface Temperature (°C)'].attrs['units'] = 'K'
+    ds['Surface Temperature (°C)'].attrs['standard_name'] = 'sea_surface_temperature'
+    ds['Surface Temperature (°C)'].attrs['long_name'] = 'Surface temperature'
+
+    ds['Frequency'].attrs['units'] = 'Hz'
+    ds['Frequency'].attrs['standard_name'] = 'wave_frequency'
+    ds['Frequency'].attrs['long_name'] = 'Frequency'
+
+    ds['Variance density'].attrs['units'] = 'm^2/Hz'
+    ds['Variance density'].attrs['standard_name'] = 'sea_surface_wave_variance_spectral_density'
+    ds['Variance density'].attrs['long_name'] = 'Spectral density'
+
+    ds['Directional spread'].attrs['units'] = 'degree'
+    ds['Directional spread'].attrs['standard_name'] = 'sea_surface_wave_directional_spread'
+    ds['Directional spread'].attrs['long_name'] = 'Directional spreading'
+    
+    ds.attrs['Conventions'] = 'CF-1.8'
+    ds.attrs['source'] = f"Sofar Spotter buoy, {filename}"
+    ds.attrs['history'] = f"generated {datetime.datetime.now().strftime('%Y-%m-%d @ %H:%M:%S')} by {getpass.getuser()}"
+    ds.attrs['references'] = 'https://content.sofarocean.com/hubfs/Technical_Reference_Manual.pdf'
+
+
+    ds = ds.rename({'Epoch Time':'time',
+                    'Frequency':'freq',
+                    'Battery Voltage (V)':'batter_voltage',
+                    'Variance density':'etfh',
+                    'Direction':'dir',
+                'Power (W)':'battery_power',
+                'Humidity (%rel)':'humidity',
+                'Significant Wave Height (m)':'Hm0',
+                'Peak Period (s)':'Tp',
+                'Mean Period (s)':'Tm',
+                'Peak Direction (deg)':'peak_dir',
+                'Peak Directional Spread (deg)':'peak_spread',
+                'Mean Direction (deg)':'mean_dir',
+                'Mean Directional Spread (deg)':'mean_spread',
+                'Directional spread':'spread',
+                'Latitude (deg)':'lat',
+                'Longitude (deg)':'lon',
+                'Wind Speed (m/s)':'wspd',
+                'Wind Direction (deg)':'wdir',
+                'Surface Temperature (°C)':'temperature'})
+    
+    ds = ds.sortby('time')
+    
+    return ds
+
+
 def read_spotter(filename):
```

Comment:
Would probably be good (and not too hard) to more elegantly integrate the added `CSV` reading capability with the existing `JSON` workflow (e.g., via a simple `if-elif` statement as discussed here: https://github.com/wavespectra/wavespectra/issues/66#issuecomment-1292680831.